### DISCUSSION TITLE
Can I extend this?

### DISCUSSION THREAD

**Discussion by khuongduy354** [SelinaDev/Godot-Roguelike-Tutorial#10] [General]
Hi, i want to extend your project, but I'm not forking it cuz I only need chapter 12, is that okay? I'll credit you, thank you!

**Comment by SelinaDev** [comment]
Shure, go ahead! I've put everything under the MIT license (except for external assets, but those are CC0), but thank you for asking. I'm glad if this project is a helpful starting point for you.

**Reply by khuongduy354** [reply]
May I ask this? There game_map.gd and procgen.gd in your repo, I saw these since part 8, but don't know what it used for?

**Reply by SelinaDev** [reply]
It took some experimenting until I settled on how to represent the map and how to generate it. Those two files are leftovers from an earlier version that I simply forgot to delete. You may ignore these and I'll delete them from the repo when I get around to it.
